### PR TITLE
Update mimir-prometheus to 75ba962

### PR DESCRIPTION
**Pull request by colega**
See: https://github.com/grafana/mimir-prometheus/pull/521

Signed-off-by: Oleg Zaytsev <mail@olegzaytsev.com>

### CODE REVIEW

File: integration/compactor_test.go [hunk 1]
```diff
@@ -229,3 +229,10 @@ func (s sample) Type() chunkenc.ValueType {
 		return chunkenc.ValFloat
 	}
 }
+
+func must[T any](v T, err error) T {
+	if err != nil {
+		panic(err)
```
Comment (pracucci) [inline integration/compactor_test.go:235]: [non blocking] I see this pattern used quite a lot in this PR. I would have preferred avoiding the panic and using `require.NoError()` instead. The `must()` could take `testing.T` in input.

Comment (pracucci) [reply integration/compactor_test.go:235]: Also, have you considered defining it in `pkg/util/test`?

**Review by pracucci — APPROVED** [review]
I'm not familiar with the native histogram changes done, so I hope I haven't missed anything. I just left a non blocking comment about `must()`.

File: integration/compactor_test.go (hunk 1 above)
Comment (colega) [reply integration/compactor_test.go:235]: I have considered both things, but they didn't convince me so I went for _little copying is better than a little dependency_.

I'll merge this as is, and then will send a separate PR with a proposal to extract it.

**Pull request merged**